upgrade/postgres.md

```diff
@@ -0,0 +1,187 @@
+Guide to upgrading PostgreSQL version `>12` to `14` using `pg_dump` and `psql`. Based on [Upgrade a PostgreSQL database with docker.](https://hollo.me/devops/upgrade-postgresql-database-with-docker.html)
```

Comment:
I've added versions info here.

Reply:
> Guide to upgrading PostgreSQL version `>12` to `14`

Hmm this is a bit confusing if I'm currently running 12. Shouldn't it be `version >= 12 to 14`?

Reply:
Fixed.